cluster/gce/gci/master.yaml

```diff
@@ -74,6 +74,33 @@ write_files:
       [Install]
       WantedBy=kubernetes.target
 
+  - path: /etc/systemd/system/kube-logrotate.timer
```

Comment:
Does this systemd job provide any log? If so, please update https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/configure.sh#l31-37